rpcs3/util/media_utils.cpp

```diff
@@ -699,7 +699,7 @@ namespace utils
 			AVCodecID used_codec = static_cast<AVCodecID>(m_video_codec_id);
 
 			// Find specified codec first
-			if (AVCodec* encoder = avcodec_find_encoder(used_codec); !!encoder)
+			if (const AVCodec* encoder = avcodec_find_encoder(used_codec); !!encoder)
```

Comment:
Well if it causes such problems on a multiplaform scale may use const_cast to remove const and thus avoiding issues in other places.